src/BloomBrowserUI/bookEdit/js/synphonyApi.ts

```diff
@@ -27,13 +27,13 @@ class SynphonyApi {
       lang_data.addGrapheme(this.source.letters.split(' '));
       lang_data.addWord(this.source.moreWords.split(' '));
       lang_data.LanguageSortOrder = this.source.letters.split(' ');
+    }
```

Comment:
If you are using the "Letters with Sight Words" option but have not entered any Letters, the stages will use just the sight words to determine decodability. It works the same as the "Allowed Words" option but uses the sight words instead of word list files.